qemu/tests/cfg/luks_image_over_qsd.cfg

```diff
@@ -0,0 +1,37 @@
+- luks_image_over_qsd:
+    only Linux
+    type = luks_image_over_qsd
+    sleep_time = 5
+    start_vm = no
+    images += " nbd1"
+#### Declare images attributes
+    image_name_stg1 = images/stg1
+    image_format_stg1 = luks
+    image_size_stg1 = 5G
+    image_secret_stg1 = redhat
+    force_create_image_stg1 = yes
```

Comment:
remove this line, force_create_image takes effect only when stg1 is defined in images(e.g. images += ' stg1'), then in the preprocess, vt tries to create the image